libs/core/langchain_core/_api/deprecation.py

```diff
@@ -271,28 +271,40 @@ def finalize(wrapper: Callable[..., Any], new_doc: str) -> T:
             class _DeprecatedProperty(property):
                 """A deprecated property."""
 
-                def __init__(self, fget=None, fset=None, fdel=None, doc=None):  # type: ignore[no-untyped-def]
+                def __init__(
+                    self,
+                    fget: Callable[[Any], Any] | None = None,
+                    fset: Callable[[Any, Any], None] | None = None,
+                    fdel: Callable[[Any], None] | None = None,
+                    doc: str | None = None,
+                ) -> None:
                     super().__init__(fget, fset, fdel, doc)
                     self.__orig_fget = fget
                     self.__orig_fset = fset
                     self.__orig_fdel = fdel
 
-                def __get__(self, instance, owner=None):  # type: ignore[no-untyped-def]
+                def __get__(
+                    self, instance: Any, owner: Union[type, None] = None
+                ) -> Any:
                     if instance is not None or owner is not None:
                         emit_warning()
+                    if self.fget is None:
```

Comment:
There was probably a bug here.
Other possibility could be to call `super().__get__` ?